src/routes/UserSearchContainer.js

```diff
@@ -35,7 +35,7 @@ class UserSearchContainer extends React.Component {
       records: 'users',
       resultOffset: '%{resultOffset}',
       perRequest: 100,
-      path: 'users',
+      path: 'users?unused=%{usersReloadToggle}',
```

Comment:
@elsenhans this should not be required. The mutations should refresh corresponding resources automatically. The refresh can be fined tuned with `shouldRefresh` if required in a similar fashion to:

https://github.com/folio-org/ui-users/blob/74220d574ab43d24fe9c3d654de665c3ee7dcca2/src/routes/LoanDetailContainer.js#L31

Reply:
Thanks! Your solution is much better.